feat: migratePresets (#10111)
renovatebot/renovate: docs/usage/self-hosted-configuration.md

## logFileLevel

## migratePresets

> Use this is you have repositories extending a particular preset but it has been removed or renamed.

Reviewer: This is confusing. Typo?

> Use this is you have repositories extending a particular preset but it has been removed or renamed.
> This is especially handy if you have a large number of repositories extending a particular preset which you want to rename, without the hassle of updating every repository individually.
> Use an empty string to indicate that the preset should be ignored rather than replaced.

Reviewer 2: ```suggestion
Use this if you have repositories that extend from a particular preset, which has now been renamed or removed.
This is handy if you have a large number of repositories that all extend from a particular preset which you want to rename, without the hassle of manually updating every repository individually.
Use an empty string to indicate that the preset should be ignored rather than replaced.
```

Maybe something like this is clearer?

Example:

```js
modules.exports = {
  migratePresets: {
    '@company': 'local>org/renovate-config',
  },
};
```

> In the above example any reference to `@company` preset will be replaced with `local>org/renovate-config`.

Reviewer 2: ```suggestion
In the above example any reference to the `@company` preset will be replaced with `local>org/renovate-config`.
```
